tests/std/tests/P0429R9_flat_map/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\concepts_latest_matrix.lst

```

Comment:
```suggestion
RUNALL_INCLUDE ..\strict_concepts_latest_matrix.lst
```
We should use the "strict" matrix at this moment. Hidden friend `iter_move` and the `iter_move` CPO are problematic in permissive modes.